proto/ptf/ptf_runner.py

```diff
@@ -117,7 +117,7 @@ def run_test(config_path, p4info_path, grpc_addr, device_id,
         os.environ['PYTHONPATH'] += ":" + pypath
     else:
         os.environ['PYTHONPATH'] = pypath
-    for iface_idx, iface_name in port_map.items():
+    for iface_idx, iface_name in list(port_map.items()):
```

Comment:
no need for that change, the 2to3 tool is being conservative here but this transformation is not needed for iteration cases

Reply:
removed that part of the changes.